lib/requirements.txt

```diff
@@ -1,15 +1,41 @@
-numpy>=1.13.3  # BSD
-colorlog~=4.7.2  # MIT
-websockets~=9.1  # BSD
-requests==2.24.0  # Apache-2.0
-PyYAML~=5.4.1  # MIT
-setuptools~=54.2.0
-fastapi~=0.63.0  # MIT
-starlette~=0.13.6  # BSD
-pydantic~=1.8.1  # MIT
-tenacity~=8.0.1  # Apache-2.0
-joblib~=1.0.1  # BSD
-pandas~=1.1.5  # BSD
-six~=1.15.0  # MIT
-minio~=7.0.3  # Apache-2.0
-uvicorn~=0.14.0  # BSD
+numpy  # BSD
+colorlog  # MIT
+websockets  # BSD
+requests  # Apache-2.0
+PyYAML  # MIT
+setuptools
+starlette  # BSD
+fastapi  # MIT
+pydantic  # MIT
+retrying  # Apache-2.0
+joblib  # BSD
+pandas  # BSD
+six>=1.13.0  # MIT
+minio  # Apache-2.0
+uvicorn  # BSD
+yapf
+mypy

```

Comment:
this seems scary, since here add all dependences for all cases.

Reply:
removed most dependences